index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="dist/css/styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="/dist/css/styles.css">
```

Links starting with `/` will always point to the root directory, so they are easier to move around as your HTML structure gets more complex.